### PR TITLE
add: mobilevit large 256 i1k model

### PR DESCRIPTION
**Pull request by sayannath**
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms and Google's Privacy Policy at https://www.google.com/policies/privacy.

We check modified Markdown files for validity using [this](https://github.com/tensorflow/tfhub.dev/blob/master/.github/workflows/contributions-validator.yml) GitHub Workflow. You can execute the same checks locally by passing the file paths of modified Markdown files (relative to the `assets/docs` directory) to validator.py e.g. `python3 ./tools/validator.py google/google.md google/models/albert_base/1.md ...`.

## Results

Results are on ImageNet-1k validation set (top-1 accuracy).

|      name     | original acc@1 | keras acc@1 |
|:-------------:|:--------------:|:-----------:|
| MobileViT_XXS |      69.0      |    68.59    |
|  MobileViT_XS |      74.7      |    74.67    |
|  MobileViT_S  |      78.3      |    78.36    |

CC: @sayakpaul @gustheman

### CODE REVIEW

**Comment by gustheman** [comment]
Sayan, can you take a look on the documentation issues please

**Comment by WGierke** [comment]
Could you create the tarball archives such that the files are in the root directory of the archive, please? There's currently e.g. a `mobilevit_s_1k_256_fe` directory, which contains the SavedModel assets. The archives couldn't be loaded by TF as of now. https://www.tensorflow.org/hub/hosting#tensorflow_hub_library_protocol has some details:  
`tar -cz -f model.tar.gz --owner=0 --group=0 -C /tmp/export-model/ .`
should create such an archive. Running  
`$ tar -tf model.tar.gz`
should then output a directory structure like
```
./
./variables/
./variables/variables.data-00000-of-00001
./variables/variables.index
./assets/
./saved_model.pb
./keras_metadata.pb
```

**Comment by sayannath** [comment]
> Could you create the tarball archives such that the files are in the root directory of the archive, please? There's currently e.g. a `mobilevit_s_1k_256_fe` directory, which contains the SavedModel assets. The archives couldn't be loaded by TF as of now. https://www.tensorflow.org/hub/hosting#tensorflow_hub_library_protocol has some details: `tar -cz -f model.tar.gz --owner=0 --group=0 -C /tmp/export-model/ .` should create such an archive. Running `$ tar -tf model.tar.gz` should then output a directory structure like
> 
> ```
> ./
> ./variables/
> ./variables/variables.data-00000-of-00001
> ./variables/variables.index
> ./assets/
> ./saved_model.pb
> ./keras_metadata.pb
> ```

Cool I will try it.

**Comment by sayannath** [comment]
Afer running this command,
```
tar -tf mobilevit_s_1k_256.tar.gz
``` 

Output
```bash
mobilevit_s_1k_256/
mobilevit_s_1k_256/keras_metadata.pb
mobilevit_s_1k_256/variables/
mobilevit_s_1k_256/saved_model.pb
mobilevit_s_1k_256/assets/
mobilevit_s_1k_256/variables/variables.data-00000-of-00001
mobilevit_s_1k_256/variables/variables.index
```

CC: @WGierke

**Comment by WGierke** [comment]
This still indicates that `mobilevit_s_1k_256.tar.gz` contains a directory `mobilevit_s_1k_256`, which then contains the model assets. This couldn't be loaded by TF.

If `/tmp/mobilevit_s_1k_256` is your local directory containing files with absolute paths
```
/tmp/mobilevit_s_1k_256/keras_metadata.pb
/tmp/mobilevit_s_1k_256/saved_model.pb
...
```
you have to run e.g.
```
cd /tmp && tar -cz -f mobilevit_s_1k_256.tar.gz --owner=0 --group=0 -C /tmp/mobilevit_s_1k_256 .
```
Running
```
tar -tf /tmp/mobilevit_s_1k_256.tar.gz
```
should then only output
```
./
./variables/
./variables/variables.data-00000-of-00001
```
indicating that the files are at the top level of the archive.

**Comment by sayannath** [comment]
Got it.

**Comment by gustheman** [comment]
any updates here?

**Comment by sayannath** [comment]
@gustheman I tried it on Google Colab, it was throwing me some errors. I am figuring it out.

**Comment by gustheman** [comment]
maybe download and try local on your machine?

**Comment by sayannath** [comment]
@WGierke, I have rectified the issue. Thanks for the assistance. 

CC: @gustheman

**Review by WGierke — APPROVED** [review]
